e2e-tests/content-strings/fields/insurance/policy/index.js

```diff
@@ -280,14 +280,18 @@ export const POLICY_FIELDS = {
     [SORT_CODE]: {
       LABEL: 'Sort code',
       HINT: 'Must be 6 digits long',
+      MINIMUM: 6,
+      MAXIMUM: 6,
     },
     [ACCOUNT_NUMBER]: {
       LABEL: 'Account number',
       HINT: 'Must be between 6 and 8 digits long',
+      MINIMUM: 6,
+      MAXIMUM: 8,
     },
   },
   FINANCIAL_ADDRESS: {
     LABEL: 'Bank address',
-    MAXIMUM: 300,
+    MAXIMUM: 500,
```

Comment:
We can remove this, if the full address validation is using `fullAddress` from `shared-validation`, it's handled in there